docs/install.md

```diff
@@ -27,9 +27,9 @@
 
 ### Get source
 
-1. Install git
-2. Get source code:
-
+1. Install git using the instructions here: https://git-scm.com/download/linux 
```

Comment:
Please remove linux specific link, this link isn't helpful for users on macOS and Windows

Reply:
Completed.